platforms/android/demo/build.gradle

```diff
@@ -2,14 +2,16 @@ apply plugin: 'com.android.application'
 
 android {
   compileSdkVersion 26
-  buildToolsVersion '25.0.3'
+  buildToolsVersion '26.0.2'
```

Comment:
Android build tools 27 are available, should we switch to that?

Reply:
From the Gradle plugin [release notes](https://developer.android.com/studio/releases/gradle-plugin.html#3-0-0):

> With this update, you no longer need to specify a version for the build tools—the plugin uses the minimum required version by default. So, you can now remove the android.buildToolsVersion property.

I suggest we just remove it.